crates/orchestrator/src/lib.rs

```diff
@@ -768,8 +833,29 @@ where
         network_config.manual_start_password = env_password.ok();
     }
 
-    network_config.config.known_nodes_with_stake = vec![];
-    network_config.config.known_da_nodes = vec![];
+    if network_config.enable_registration_verification {
```

Comment:
I know this wasn't introduced in this PR, but:

What do you think about enabling registration verification automatically if the list is nonzero? And having it be off if it's zero?

Reply:
oh yes, great idea!

I'll update the PR and remove the `enable_registration_verification` field altogether